manuscript/03.results.md

```diff
@@ -64,12 +64,12 @@ As an application of our model, we used phenotypic measurements of the G1 and S/
 
 ### Lapatinib treatment leaves stable and cyclical states in the cell population
 
-Lapatinib is a EGFR/HER2 inhibitor that induces cell cycle arrest in G1 phase (CITE). AU565 cells were treated with 25 nM, 50 nM, and 250 nM of lapatinib during the experiment and imaged and tracked for over 96 hours. We fitted the lapatinib treated data to the model with 3 states, based on our BIC-based model selection (Fig. {@fig:expBIC}a). As a result of fitting, the population was classified into two fairly stable states and a cycle (Fig. {@fig:emissionsLPT}i). Lineages that belong to the same state showed that increasing the lapatinib concentration results in the mean of G1 lengths in almost all states, especially in cells of state 1 (Figs. {@fig:emissionsLPT}a-e, @fig:LapatinibTree) and almost no change in the mean of G2 lengths (Fig. {@fig:emissionsLPT}f). The probability of cell death at G1 and G2 phase was higher in state 2 and 3, respectively, but only when the concentration of the drug is more than 50 nM (Figs. {@fig:emissionsLPT}g-h). Further, cells in state 1 seem to be resistant to cell death in all concentrations, and the effect of the drug in these cells is solely on G1 phase extension (Figs. {@fig:emissionsLPT}e-h). 
```

Comment:
How did you find this reference? Does it support the statement that Lapatinib arrests cells in G1?

Reply:
It looks like a review to me, but I can't find the full paper.

Reply:
I couldn't find it either... But in google scholar, I saw it has ~160 citations so it is a real paper. I figured it might be one of those papers that need registration on some website to give access to the full text.

Reply:
You should *never* cite a paper without reading it first. For all you know, it's only cited because people made the same decision as you.

This needs to be a reference showing that Lapatinib leads to G1 arrest. You can just change it back to `CITE` to merge this?

Reply:
Actually, I'm just going to do this and then merge.